docs/detections/api/exceptions/api-update-exception-container.asciidoc

```diff
@@ -35,8 +35,11 @@ the container's `id` field is not used.
 exception containers. |No
 |`type` |String a|The type of exception, which must be one of these:
 
-* `detection`: Detection rule exception
-* `endpoint`: Endpoint alert exception
+|`type` |String a|The type of exception list, which must be one of these:
```

Comment:
```suggestion
|`type` |String a|The type of exception list, which must be one of:
```

Reply:
You could also go with "Valid values are:" which appears elsewhere in this API

Reply:
I like this ^. It's much more explicit. 

```suggestion
|`type` |String a|The type of exception list. Valid values are:
```